repository-data/application/src/main/resources/hcm-content/content/documents/lks/home.yaml

```diff
@@ -0,0 +1,94 @@
+/content/documents/lks/home:
```

Comment:
@thojava - Did you place this document under `repository-data/application` in order to push this to higher brCloud env (Test, Staging, etc) so that content authors will have a `Home` document to start with ? If not, could you move this under `repository-data/development` ?

@thojava @duncanheron - This implementation requires a document named `Home` of type `HomePage` must exists under root folder `LKS` in order for home page to pick the sections for rendering. Worth passing on to Olivia and HEE content authors ? Thanks!

Reply:
@smanivasag That should be under the `repository-data/development` module. I was assumed the autoexport do it by default. Let me update that.

There should a protocol where I can pass the requirement setup which need to be done by the editor to make the feature working on the PROD environment @duncanheron

Reply:
Thanks @thojava - Autoexport seems to update the existing ones on `repository-date/development`, but unfortunately the new ones still being exported under `repository-data/application` :(

Would you mind adding the requirement setup instructions for HomePage on our #hee-dev channel mentioning Olivia, Matt & Duncan and guess they would pick it up from there. Worth adding the instructions into the ticket comments as well.